packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -130,6 +131,7 @@ liquidator10 = "0x45556408e543158f74403e882E3C8c23eCD9f732"
 
 [var.ae_liquidators]
 ae_liquidator1 = "0xF39f72E8E8D16833601C3f1ac33835ca5C69f6E4"
+ae_liquidator2 = "0xF39f72E8E8D16833601C3f1ac33835ca5C69f6E4"
```

Comment:
second liquidator the same address ?

Reply:
yes, just to accomodate the fact that on mainnet we have 2